src/rust/vendor-rust-std.sh

```diff
@@ -0,0 +1,12 @@
+RUST_DIR="$(rustc --print=sysroot)"
```

Comment:
The name of the script sounds like it vendors only the stdlib, but it vendors all our normal deps too and should be used instead of `cargo vendor`, right?

Maybe rename the script and add a section to src/rust/README.md about how to vendor.